WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutPickerTracker.kt

```diff
@@ -3,7 +3,8 @@ package org.wordpress.android.ui.layoutpicker
 interface LayoutPickerTracker {
     fun trackPreviewModeChanged(mode: String)
 
-    fun trackThumbnailModeTapped(mode: String)
+    fun trackThumbnailModeTapped(mode: String) {
```

Comment:
Perhaps I should have left a comment here, something like:
// NOOP implementation provided to make override optional in subclasses